src/include/distributed/pg_dist_partition.h

```diff
@@ -52,6 +52,7 @@ typedef FormData_pg_dist_partition *Form_pg_dist_partition;
 #define DISTRIBUTE_BY_APPEND 'a'
 #define DISTRIBUTE_BY_HASH 'h'
 #define DISTRIBUTE_BY_RANGE 'r'
+#define DISTRIBUTE_BY_ALL 'l'
```

Comment:
We'd have to drop the existing enum, which would have to cascade to `create_distributed_table`, which might cascade to user-defined wrappers.